src/apps/frontend/pages/tasks/task-container.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react';
+
+import { HeadingLarge, VerticalStackLayout } from '../../components';
+import { useTaskContext } from '../../contexts';
+import { AsyncError } from '../../types';
+
+import TaskHeader from './task-header';
+import TaskSection from './task-section';
+
+interface TaskContainerProps {
+  onError: (error: AsyncError) => void;
+}
+
+const TaskContainer: React.FC<TaskContainerProps> = ({ onError }) => {
```

Comment:
No need for this, as this is not being reused.

Inline this directly in `const Tasks: React.FC = () => {`

Reply:
Sure